enclave.go

```diff
@@ -1004,40 +1009,41 @@ func (e *Enclave) DeleteIdentity(ctx context.Context, identity Identity) error {
 //
 // The pattern matching happens on the server side. If pattern is empty
 // ListIdentities returns all identities.
-func (e *Enclave) ListIdentities(ctx context.Context, pattern string) (*IdentityIterator, error) {
+func (e *Enclave) ListIdentities(ctx context.Context, prefix string, n int) ([]Identity, string, error) {
 	const (
-		APIPath  = "/v1/identity/list"
-		Method   = http.MethodGet
-		StatusOK = http.StatusOK
+		APIPath         = "/v1/identity/list"
+		Method          = http.MethodGet
+		StatusOK        = http.StatusOK
+		MaxResponseSize = 1 * mem.MiB
 	)
+	type Response struct {
+		Names      []Identity `json:"names"`
```

Comment:
Should we call this field as `Identities` ?

Reply:
Yeah, we can do this. This is probably copy-pasted from `ListKeys`, ListPolicies` etc.